.github/workflows/cmd.yml

```diff
@@ -0,0 +1,417 @@
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: `Sorry, only ${{ github.event.repository.owner.login }} members can run commands.`
+            })
+
+  acknowledge:
+    needs: is-org-member
+    if: ${{ startsWith(github.event.comment.body, '/cmd') && needs.is-org-member.outputs.member == 'true' }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Add reaction to triggered comment
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            github.rest.reactions.createForIssueComment({
+              comment_id: ${{ github.event.comment.id }},
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              content: 'eyes'
+            })
+
+  clean:
+    needs: is-org-member
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Clean previous comments
+        if: ${{ startsWith(github.event.comment.body, '/cmd') && contains(github.event.comment.body, '--clean') && needs.is-org-member.outputs.member == 'true' }}
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            github.rest.issues.listComments({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo
+            }).then(comments => {
+              for (let comment of comments.data) {
+                console.log(comment)
+                if (
+                  ${{ github.event.comment.id }} !== comment.id &&
+                    (
+                      (
+                        (
+                          comment.body.startsWith('Command') || 
+                          comment.body.startsWith('<details><summary>Command') || 
+                          comment.body.startsWith('Sorry, only ')
+                        ) && comment.user.type === 'Bot'
+                      ) ||
+                      (comment.body.startsWith('/cmd') && comment.user.login === context.actor)
+                    )
+                ) {
+                  github.rest.issues.deleteComment({
+                    comment_id: comment.id,
+                    owner: context.repo.owner,
+                    repo: context.repo.repo
+                  })
+                }
+              }
+            })
+  help:
+    needs: [ clean, is-org-member ]
+    if: ${{ startsWith(github.event.comment.body, '/cmd') && contains(github.event.comment.body, '--help') && needs.is-org-member.outputs.member == 'true' }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Get command
+        uses: actions-ecosystem/action-regex-match@v2
+        id: get-pr-comment
+        with:
+          text: ${{ github.event.comment.body }}
+          regex: '^(\/cmd )([\s\w-]+)$'
```

Comment:
Maybe we could add here some check that this is a valid command.

Reply:
As I understand the argparse will take care of it and validate upon the input requirements. So I should pass `/cmd blabalb` but return validation error

Reply:
![image](https://github.com/user-attachments/assets/4edbe47f-c9c2-4b14-a1f0-e826150a6126)
![image](https://github.com/user-attachments/assets/7fd43ac0-d04f-44e3-b1c3-0c50bc03f8e3)